tests/acceptance/bashunit_direct_fn_call_test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+function set_up_before_script() {
+  TEST_ENV_FILE="tests/acceptance/fixtures/.env.default"
+}
+
+function test_bashunit_direct_fn_call_passes() {
+  local expected="foo"
+  local actual="foo"
+
+  ./bashunit -a assert_equals --env "$TEST_ENV_FILE" "$expected" $actual
+  assert_successful_code
+}
+
+function test_bashunit_direct_fn_call_failure() {
+  local expected="foo"
+  local actual="bar"
+
+  assert_match_snapshot "$(./bashunit -a assert_equals --env "$TEST_ENV_FILE" "$expected" $actual)"
+  assert_general_error "$(./bashunit -a assert_equals --env "$TEST_ENV_FILE" "$expected" $actual)"
```

Comment:
since we already use a `-a` or `--assert` as cli option, wdyt about leaving the `assert_` prefix out of the argument...?

would be less to write and less duplication

```suggestion
  assert_match_snapshot "$(./bashunit -a equals --env "$TEST_ENV_FILE" "$expected" $actual)"
  assert_general_error "$(./bashunit -a equals --env "$TEST_ENV_FILE" "$expected" $actual)"
```

Reply:
What about allowing both? It cost nothing really, and some developers might be purist in the sense they want to use the complete actual assert function (with the prefix), and some (like you) might prefer without `assert_` prefix to avoid "duplication". Implementing this is pretty easy, just when checking if the function exists, in case it does not, then try again prepending the prefix

Reply:
Having both would also be great, thanks